### ISSUE TITLE
needs to be updated for request 2.51

### ISSUE THREAD

**Issue by coolaj86** [coolaj86/jarson#3]
Currently you must use request@2.44.0 in order to use jarson. (this addresses issue #1 and #2.)

``` bash
npm install --save request@2.44.0
```

Somewhere between 2.44.0 and 2.51.0 breaking changes were introduced to the internals of the cookie storage mechanism (as a result of discussion on the github issues about the very code that lead to this module).

Unfortunately, it was updated with the intent of saving cookies to a file rather than transferring them as json, so I don't know how convoluted it will be to rework this module (i.e. creating a custom memory-store plugin for request's new cookie system).

That said, I probably won't update jarson until I hear about or experience a case where 2.44 doesn't meet the needs of a use case just as well as a newer version.

**Comment by jlank** [comment]
Do you have an ETA for when this will land?  Would love to use it :)

**Comment by coolaj86** [comment]
Nope, but you can use request 2.44.0 and use it now. I don't know what's new in 2.51, but I can't imagine it's anything super important (I imagine it's mostly just the breaking changes to the way cookies are stored).

**Comment by jlank** [comment]
will do, thanks!
